keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1635,7 +1641,7 @@ conditions apply:
   Filter
 
 One of the ResourceClaims satisfying these criteria is picked randomly and deallocation
-is requested by setting the Deallocate field. The scheduler then needs to wait
+is requested by setting the ResourceClaimStatus.DeallocationRequested field. The scheduler then needs to wait
 for the resource driver to react to that change and deallocate the resource.
 
 This may make it possible to run the Pod
```

Comment:
What will be the order of this plugin compared to preemption?

Reply:
This operation here is triggered by observing that it was some allocated claim which prevented pod scheduling. I think this deallocate+wait then should happen before preemption is attempted.

Would preemption be triggered at all in this case? As far as the scheduler knows, all nodes where rejected by the dynamic resource allocation plugin. Why should it evict pods from any of them when those pods where not the reason that the new pod cannot run.

Reply:
> Why should it evict pods from any of them when those pods were not the reason that the new pod cannot run?

What if one of these pods is using the ResourceClaim that the new pod needs?

Reply:
I missed this comment earlier. The answer is that taking away a claim from a pod is not supported yet, even if a more important pod is blocked by a less important pod. Support this will have to be planned and added later.